CautionaryAlertsApi/V1/Helpers/GoogleSheetHelpers.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Sheets.v4.Data;
+
+namespace CautionaryAlertsApi.V1.Helpers
+{
+    public static class GoogleSheetHelpers
+    {
+        public static IEnumerable<IEnumerable<T>> PivotRows<T>(this IEnumerable<IEnumerable<T>> source)
+        {
+            var enumerators = source.Select(e => e.GetEnumerator()).ToArray();
+
+            try
+            {
+                while (enumerators.All(e => e.MoveNext()))
+                    yield return enumerators
+                        .Select(e => e.Current)
+                        .ToArray();
+            }
+            finally
+            {
+                Array.ForEach(enumerators, e => e.Dispose());
+            }
```

Comment:
You've a try with no catch here - is the intention to swallow all exceptions? If so, i'd make that explicit